packages/platform/src/App.tsx

```diff
@@ -9,30 +9,37 @@ import { UsjNodeOptions } from "shared-react/nodes/scripture/usj/usj-node-option
 import { getViewOptions } from "./editor/adaptors/view-options.utils";
 import { formattedViewMode as defaultViewMode } from "./editor/plugins/toolbar/view-mode.model";
 import ViewModeDropDown from "./editor/plugins/toolbar/ViewModeDropDown";
-import Editor from "./editor/Editor";
+import Editor, { EditorRef } from "./editor/Editor";
 import "./App.css";
 
+const defaultUsj = usxStringToUsj('<usx version="3.0" />');
 const defaultScrRef: ScriptureReference = { /* PSA */ bookNum: 19, chapterNum: 1, verseNum: 1 };
-
-const usj = usxStringToUsj(usx);
-
 const nodeOptions: UsjNodeOptions = { [immutableNoteCallerNodeName]: { onClick: () => undefined } };
 
-const onChange = (usj: Usj) => console.log({ usj });
-
 export default function App() {
+  const editorRef = useRef<EditorRef>(null);
   const [viewMode, setViewMode] = useState(defaultViewMode);
   const [scrRef, setScrRef] = useState(defaultScrRef);
   const viewOptions = useMemo(() => getViewOptions(viewMode), [viewMode]);
 
+  setTimeout(() => {
+    editorRef.current?.setUsj(usxStringToUsj(usx));
+  }, 1000);
+
+  const onChange = useCallback((usj: Usj) => {
+    console.log({ usj });
+    editorRef.current?.setUsj(usj);
```

Comment:
I could be misunderstanding, but I thought the point of making it uncontrolled was so you didn't need to set the usj when you received notice of change from the editor. Shouldn't we remove this line?